roadmaps/2022.md

```diff
@@ -70,13 +70,44 @@ Regarding this point, more details are available on the product team roadmap, bu
 
 ## Q2 2022
 
-- [ ] Finish the recruitment process to hire an intern. Depending on the intern, the onboarding will be in Q2 or later.
-- [ ] Release v0.27.0 including indexation speed improvements, the nested field support, the typo tolerance customization, the stabilization/improvement of the cropping & highlighting.
-- [ ] Keep being more and more transparent regarding the pre-release tasks each developer makes: write down regularly on GitHub the context, the hypotheses, the results & conclusions, the graphs, the metrics...
-- [ ] Keep improving the indexation speed: work with constraints, add benchmarks to prove the change is valuable...
-- [ ] v0.28.0: stabilize the API routes. v0.28.0 will be released at the beginning of Q3 but the specs and the whole sprint is during Q2.
-- [ ] Keep working on the tokenizer
-  - [ ] Clarify the organization regarding the issue management
-  - [ ] Make it more contribuable in order to let the community add their language support easily
-- [ ] Autobatching feature: stabilize it, or at least start to work on the stabilization.
-- [ ] Automate our release process by deploying releasemops 🤖
+- [X] Finish the recruitment process to hire an intern. Depending on the intern, the onboarding will be in Q2 or later.
+- [X] Start process to hire a senior software engineer.
+- [X] Release v0.27.0 including indexation speed improvements, the nested field support, the typo tolerance customization, the stabilization/improvement of the cropping & highlighting.
+- [X] Keep being more and more transparent regarding the pre-release tasks each developer makes: write down regularly on GitHub the context, the hypotheses, the results & conclusions, the graphs, the metrics...
+- [X] Keep improving the indexation speed: work with constraints, add benchmarks to prove the change is valuable...
+- [X] v0.28.0: stabilize the API routes. v0.28.0 will be released at the beginning of Q3 but the specs and the whole sprint is during Q2.
+- [X] Keep working on the tokenizer
+  - [X] Clarify the organization regarding the issue management
+  - [X] Make it more contribuable in order to let the community add their language support easily
+  - [X] Publish the tokenizer to crates.io
+- [ ] *Started in Q2* - Autobatching feature: stabilize it, or at least start to work on the stabilization.
+- [ ] *Delayed* Automate our release process by deploying releasemops 🤖
+
+## Q3 2022
+
+### Team
+
+- [ ] Finish the recruitment process to hire a senior software engineer.
+- [ ] Onboard the new software engineer.
+- [ ] Adapt the team synchronization and rhythm according to the Product Squad (PS) freshly introduced by the product team.
+
+### Road to v1
+
+- [ ] Stabilize the auto-batching feature: this feature is activated by default and cannot be disabled.
```

Comment:
> "this feature is activated by default and cannot be disabled." 

There is no need to specify this information perhaps. Nothing is sure yet but the ideal solution would be to release the auto-batching so that it works in a completely transparent way. Maybe it will still be possible to disable it.

Reply:
> Why wouldn't it be possible to disable it

As a first step maybe we would like to put auto-batching by default, and let the possibility to disable it if a big bug is present to avoid friction for the users. We could say "sorry for the bug we are fixing it, in the meantime you can disable auto-batching like this to remove the bug"

> are we confident enough?

The goal is to be confident enough to remove the autobatching setting (enable or disable it) because we know the feature is stable enough.